granian/log.py

```diff
@@ -37,12 +37,18 @@ class LogLevels(str, Enum):
     'handlers': {'console': {'formatter': 'generic', 'class': 'logging.StreamHandler', 'stream': 'ext://sys.stdout'}},
 }
 
-logger = logging.getLogger()
+# NOTE: to be consistent with the Rust module logger name
+logger = logging.getLogger("_granian")
 
 
-def configure_logging(level: LogLevels, config: Optional[Dict[str, Any]] = None):
+def configure_logging(level: LogLevels, config: Optional[Dict[str, Any]] = None, enabled: bool = True):
     log_config = copy.deepcopy(LOGGING_CONFIG)
+
     if config:
         log_config.update(config)
+
     log_config['root']['level'] = log_levels_map[level]
```

Comment:
@Aeron should this also be changed to `'_granian'`?

Reply:
Oh, I got carried away and forgot to answer. And so it’s merged already. My apologies.

I think we don’t need to do it since we’re configuring the root logger anyway, so the Granian logger inherits the logging level. But I have a few suspicions about the difference between in-Python and in-Rust Granian loggers, so I’ll explicitly test it later and fix it if necessary.

Reply:
No worries. I made the change myself and avoid to tamper with the root logger seems to work!